.github/workflows/prod-cd.yml

```
@@ -0,0 +1,54 @@
+          workflow: prod-ci.yml          # CI 파일 이름
+          workflow_conclusion: success
+          branch: prod
+          name: app-jar                  # prod-ci에서 업로드한 이름과 동일
+          path: ./artifact
+
+      - name: Check downloaded files
+        run: ls -R .
+
+      # 2) JAR & 배포 스크립트 EC2로 전송
+      - name: Copy JAR & deploy script to EC2
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2012:info:1:12: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)